DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -0,0 +1,133 @@
+# script to set extra env variables
+# taking some stuff from alien
+
+# process flags passed to the script
+
+export SETENV_NO_ULIMIT=1
+
+# to avoid memory issues
+export DPL_DEFAULT_PIPELINE_LENGTH=16
+
+# detector list
+export WORKFLOW_DETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,HMP,EMC,PHS,CPV
+
+# ad-hoc settings for CTF reader: we are on the grid, we read the files remotely
+echo "*********************** mode = ${MODE}"
+unset ARGS_EXTRA_PROCESS_o2_ctf_reader_workflow
+if [[ $MODE == "remote" ]]; then
+  export INPUT_FILE_COPY_CMD="\"alien_cp ?src file://?dst\""
+  export ARGS_EXTRA_PROCESS_o2_ctf_reader_workflow="--remote-regex \"^alien:///alice/data/.+\""
```

Comment:
This anyway default, and not needed.
But, once the https://github.com/AliceO2Group/AliceO2/pull/9348 is merged, we should add instead
```
export ARGS_EXTRA_PROCESS_o2_ctf_reader_workflow="--apply-trigger-offset"
```
to correct TRD shift.

Reply:
Here for now I won't change, in case we start a test run for the throttling and your PR is not merged yet.

Reply:
In any case, we can also take the setenv_extra from input files.